tests/auto/rt_auto.sh

```diff
@@ -8,38 +8,40 @@ else
   exit 1
 fi
 
-export RT_COMPILER='intel'
-source ../detect_machine.sh
-echo "Machine ID: "+$MACHINE_ID
-if [[ $MACHINE_ID = hera.* ]]; then
+if [[ $HOSTNAME == hfe* ]]; then
+  MACHINE_NAME=hera
   WORKDIR=/scratch1/NCEPDEV/nems/Brian.Curtis/test
   export PATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/bin:$PATH
   export PYTHONPATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
-elif [[ $MACHINE_ID = orion.* ]]; then
+elif [[ $HOSTNAME == Orion-login-* ]]; then
```

Comment:
Just use Orion* to be on the safe side?

Reply:
I don't know of an Orion login that didn't hat that format, unless you had issues? I'll keep this as is.